test/LetsEncrypt.Azure.UnitTests/AzureKeyVaultTests.cs

```diff
@@ -19,6 +25,27 @@ namespace LetsEncrypt.Azure.UnitTests
 {
     public class AzureKeyVaultTests
     {
+        private X509Certificate2 BuildSelfSignedServerCertificate(string certificateHost)
+        {
+            var CN = certificateHost.Replace("https://", ""); // Sanitize from certificate common name, dns name and friendly name
+
+            var sanBuilder = new SubjectAlternativeNameBuilder();
+            sanBuilder.AddIpAddress(IPAddress.Loopback);
+            sanBuilder.AddIpAddress(IPAddress.IPv6Loopback);
```

Comment:
Interesting addition. Any particular reason you added this to the SAN?

By the way, there is similar code in https://github.com/natemcmaster/LetsEncrypt/blob/a62c5cea0b0f525919e0b8fc0318119e662cd99e/test/LetsEncrypt.UnitTests/TestUtils.cs#L13. Maybe we could import that into this project to avoid duplication?

Reply:
Ipaddress allows us in theory to test it on loopback IP but it's not intentional and not that useful, it snuck in there from something i based my code on. I'll drop this entirely and use yours from utils.

Reply:
This required that the Azure test reference the main tests, so i did that.